assignment/impl/src/test/org/sakaiproject/assignment/impl/AssignmentServiceTest.java

```diff
@@ -964,14 +964,15 @@ public void submissionStatus() {
         String submitterId = UUID.randomUUID().toString();
 
         // test for non existent submission
-        String status = assignmentService.getSubmissionStatus("SUBMISSION_DOESNT_EXIST");
+        String status = assignmentService.getSubmissionStatus("SUBMISSION_DOESNT_EXIST", true);
+        String statusWithDate = assignmentService.getSubmissionStatus("SUBMISSION_DOESNT_EXIST", false);
```

Comment:
I believe you don't need this here (but you'll need the variable definition later on)